### PR TITLE
Add DTSTAMP field to entries

### LINKED ISSUE #15: Sync issue with darwin calendar server
This issue was moved from the issue tracker at GitLab.

---

I am trying creating a todo and then syncing it with darwin calendar server.
That fails with following error from darwin calendar server:

    2016-04-15 02:55:17+0300 [-] [caldav-1]  [-] [twistedcaldav.method.put_common#error] Invalid calendar data: Calendar data had unfixable problems:
    2016-04-15 02:55:17+0300 [-] [caldav-1]           [VTODO] Missing or too many required property: DTSTAMP

What I did:
1. I installed current version of todoman with pip.
1. Set up sync with darwin calendar server with vdirsyncer. Setup works (I can sync calendar/todo items, edit todos created from other sources, etc).
1. Tried to create new todo with following line: __todo new -l calendar "test todo"__
1. Todo is created locally and can be seen/modified
1. Sync fails with quoted error in server logs.

Resulting file (and file with same summary, but created from android "tasks" application, that syncs successfully are attached.
[fail.ics](/uploads/71ee030b4c82d2a875d43bd0b2e9953c/fail.ics)
[success.ics](/uploads/2208fb32e07c21c549f0aa2fdd75f586/success.ics)

The successful one does contain DTSTAMP field.

### PR DESCRIPTION
**Pull request by hobarrera**
Fixes #15

### CODE REVIEW

**Comment by hobarrera** [comment]
This is ready to merge. I'll leave it open for a short while in case anybody has any feedback.

**Comment by fhunter** [comment]
Checked on my setup. It works.

File: tests/test_basic.py [hunk 1]
```diff
@@ -105,5 +107,22 @@ def test_delete(tmpdir, runner, create):
...
+def test_dtstamp(tmpdir, runner, create):
+    """
+    Test that we add the DTSTAMP to new entries as per RFC5545.
+    """
+    result = runner.invoke(cli, ['new', '-l', 'default', 'test event'])
+    assert not result.exception
+
+    todo_file = (tmpdir + '/default').listdir()[0]
+    # Trailing Z means it's UTC, which is what the spec requires.
+    assert re.search(
```
Comment (untitaker) [inline tests/test_basic.py]: Why not use icalendar to parse?

Comment (hobarrera) [inline tests/test_basic.py]: Slipped my mind. Fixed.

**Comment by untitaker** [comment]
LGTM.

**Pull request merged**